op-mode-definitions/show-isis.xml

```diff
@@ -0,0 +1,182 @@
+<?xml version="1.0"?>
+<interfaceDefinition>
+  <node name="show">
+    <children>
+      <node name="isis">
+        <children>
+          <node name="database">
+            <properties>
+              <help>Show IS-IS link state database</help>
+            </properties>
+            <children>
+              <leafNode name="detail">
+                <properties>
+                  <help>Show detailed information</help>
+                </properties>
+                <command>/usr/bin/vtysh -c "show isis database detail"</command>
+              </leafNode>
+            </children>
+            <command>/usr/bin/vtysh -c "show isis database"</command>
+          </node>
+          <tagNode name="database">
+            <properties>
```

Comment:
Can we have completion helper list here?
```
      <completionHelp>
        <list>lsp-id detail</list>
      </completionHelp>
```

Reply:
Absolutely :)